tests/elements/radio_button/test_radio_button.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+from pages_methods.elements.radio_button import RadioButton
+from pages_methods.main_page_demoqa import MainPageDemoQA
+
+
+@pytest.mark.usefixtures
+class TestRadioButton:
+
+    def test_select_elements(self):
+        assert MainPageDemoQA().check_elements_is_visible(self.driver)
+        MainPageDemoQA().click_elements(self.driver)
```

Comment:
de ce assert inainte si apoi click ?! daca nu gaseste elementul in click nu oricum pica?